packages/pancake-uikit/src/theme/types.ts

```diff
@@ -29,6 +29,11 @@ export type Shadows = {
 
 export type Gradients = {
   bubblegum: string;
+  cardBackground: {
+    default: string;
+    blue: string;
+    violet: string;
+  };
```

Comment:
The `Gradients` object should only be top-level values, i.e. no `cardBackground`. That means however we will need to come up with a name for `default`

Reply:
Yea makes sense not to nest them. Let me ping Bugs for some inspiration on 'default'

Reply:
Updated this to remove nesting & reflect Figma color scheme